tasks/pyproject.toml

```diff
@@ -19,13 +19,14 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "^3.10"
```

Comment:
This is required because of the newly added `robocorp-truststore` below.

Reply:
This will effectively make our minimum supported version 3.10 for our framework... so, just double-checking if this is Ok at this point.

Reply:
Yes, I think this is not ok at all, it will attract these bumps everywhere dependent, and may affect our pipelines as well in case we though to test Py3.9.

But I'm totally up to jump to Py3.10 at minimum since `truststore` doesn't work below that. We should discuss first, maybe with @mikahanninen as well in case we'll ever depend on stuff from **robocorp** as we've seen with **robocorp-storage**.

Reply:
Since `truststore` was designed to be brought by Conda, I won't interfere with, not even under our dev deps, as this will require jumping to Py3.10 which might bring more trouble than value for now.

Still, as a workaround for the missing `truststore` issuing warnings, will add it under _devutils/requirements.txt_ so the parent env out of which the Poetry ones will derive will make it available.